packages/p2p-v2/src/components/Modals/MyAdsDeleteModal/MyAdsDeleteModal.tsx

```diff
@@ -0,0 +1,95 @@
+import React, { memo, useEffect } from 'react';
+import clsx from 'clsx';
+import Modal from 'react-modal';
+import { useDevice } from '@/hooks';
+import { p2p } from '@deriv/api';
+import { Button, Text } from '@deriv-com/ui';
+import { customStyles } from '../helpers';
+
+type TMyAdsDeleteModalProps = {
+    error?: string;
+    id: string;
+    isModalOpen: boolean;
+    onClickDelete: () => void;
+    onRequestClose: () => void;
+};
+
+const MyAdsDeleteModal = ({ error, id, isModalOpen, onClickDelete, onRequestClose }: TMyAdsDeleteModalProps) => {
+    const { data: advertInfo, isLoading: isLoadingInfo } = p2p.advert.useGet({ id });
+    const { isMobile } = useDevice();
+    const textSize = isMobile ? 'md' : 'sm';
+
+    const hasActiveOrders = advertInfo?.active_orders && advertInfo?.active_orders > 0;
+
+    useEffect(() => {
+        Modal.setAppElement('#v2_modal_root');
```

Comment:
I suggest not to hardcode this all over the code but that's something the team can work on in a separate card.